stock_inventory_discrepancy/models/stock_inventory_line.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=import-error,protected-access,too-few-public-methods
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+"""stock_inventory_line"""
+
+from odoo import api, fields, models
+
+
+class StockInventoryLine(models.Model):
+    """Class to inherit stock.inventory.line"""
+    _inherit = 'stock.inventory.line'
+
+    @api.multi
+    def _compute_discrepancy(self):
+        """Function to compute discrepancy"""
+        for line in self:
+            line.discrepancy_qty = line.product_qty - line.theoretical_qty
+            if line.theoretical_qty:
+                line.discrepancy_percent = 100 * abs(
+                    (line.product_qty - line.theoretical_qty) /
+                    line.theoretical_qty)
+            elif not line.theoretical_qty and line.product_qty:
+                line.discrepancy_percent = 100.0
+
+    @api.multi
+    def _compute_discrepancy_threshold(self):
+        """Function to compute discrepancy threshold"""
+        for line in self:
+            whs = line.location_id.get_warehouse()
+            if line.location_id.discrepancy_threshold > 0.0:
+                line.discrepancy_threshold = line.location_id.\
+                    discrepancy_threshold
+            elif whs.discrepancy_threshold > 0.0:
+                line.discrepancy_threshold = whs.discrepancy_threshold
+            else:
+                line.discrepancy_threshold = False
+
+    discrepancy_qty = fields.Float(
```

Comment:
Please put fields on top and declare compute functions with ''.

https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#models